docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -17,12 +17,22 @@ YugabyteDB Anywhere first needs to be installed on a host computer, and then you
 
 You can install YugabyteDB Anywhere on the following Linux distributions:
 
-- Ubuntu 16.04, 18.04, or 20.04 LTS
-- Red Hat Enterprise Linux (RHEL) 7.x
-- Oracle Linux 7.x
-- CentOS 7.x
-- Amazon Linux (AMI) 2014.03, 2014.09, 2015.03, 2015.09, 2016.03, 2016.09, 2017.03, 2017.09, 2018.03, or 2.0
-- Other [operating systems supported by Replicated](https://www.replicated.com/docs/distributing-an-application/supported-operating-systems/)
+- CentOS (default)
+- Ubuntu 18 and 20, via Replicated.
+- AlmaLinux OS 8, via Replicated and Yugabundle.
```

Comment:
Currently on CentOS 7 is Yugabundle. Everything else is Replicated. I'm not even sure if we need to call out Yugabundle here given it is not publicly documented AND it's getting deprectaed in the near future in favor of yba-installer.

Reply:
Deleted Yugabundle